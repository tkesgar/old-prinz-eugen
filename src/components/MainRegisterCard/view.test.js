import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import ViewComponent from './view'

describe('CardRegister', () => {
  const wrapper = mount(ViewComponent, { localVue })

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
