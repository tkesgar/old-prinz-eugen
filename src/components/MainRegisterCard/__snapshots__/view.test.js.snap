// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardRegister should match snapshot 1`] = `
<div
  class="card register-card mx-auto"
>
  <!---->
  <!---->
  <div
    class="card-body"
  >
    <h4
      class="card-title"
    >
      Daftar pengguna baru
    </h4>
    <!---->
    <form
      class=""
    >
      <div
        class="form-group"
        id="__BVID__4"
        role="group"
      >
        <label
          class="d-block"
          for="RegisterCard_name"
          id="__BVID__4__BV_label_"
        >
          Nama pengguna
        </label>
        <div>
          <input
            aria-required="true"
            class="form-control"
            id="RegisterCard_name"
            pattern="^\\\\w{1,16}$"
            placeholder="Nama pengguna"
            required="required"
            type="text"
          />
           
          <small
            class="form-text text-muted"
          >
            
        Nama pengguna hanya terdiri dari huruf, angka, dan garis bawah (_), dengan panjang
        maksimal 16 karakter.
      
          </small>
          <!---->
          <!---->
          <!---->
        </div>
      </div>
       
      <div
        class="form-group"
        id="__BVID__6"
        role="group"
      >
        <label
          class="d-block"
          for="RegisterCard_email"
          id="__BVID__6__BV_label_"
        >
          Alamat email
        </label>
        <div>
          <input
            aria-required="true"
            class="form-control"
            id="RegisterCard_email"
            placeholder="Email"
            required="required"
            type="email"
          />
           
          <small
            class="form-text text-muted"
          >
            
        
        Kami tidak menyimpan alamat email Anda dalam bentuk utuh.
      
          </small>
          <!---->
          <!---->
          <!---->
        </div>
      </div>
       
      <div
        class="form-group"
        id="__BVID__8"
        role="group"
      >
        <label
          class="d-block"
          for="RegisterCard_password"
          id="__BVID__8__BV_label_"
        >
          Kata sandi
        </label>
        <div>
          <input
            aria-required="true"
            class="form-control"
            id="RegisterCard_password"
            minlength="8"
            placeholder="Kata sandi"
            required="required"
            type="password"
          />
           
          <small
            class="form-text text-muted"
          >
            
        Panjang kata sandi minimal adalah 8 karakter.
      
          </small>
          <!---->
          <!---->
          <!---->
        </div>
      </div>
       
      <fieldset
        class="form-group"
        id="__BVID__10"
      >
        <!---->
        <div
          role="group"
          tabindex="-1"
        >
          <div />
          <!---->
          <!---->
          <!---->
        </div>
      </fieldset>
       
      <button
        class="btn btn-secondary btn-block"
        type="submit"
      >
        Daftar
      </button>
    </form>
  </div>
  <!---->
  <!---->
</div>
`;
